services/kube-prometheus-stack/48.3.2/helmrelease/kube-prometheus-stack.yaml

```diff
@@ -11,7 +11,7 @@ spec:
         kind: HelmRepository
         name: mesosphere.github.io-charts-staging
         namespace: kommander-flux
-      version: 46.8.0
+      version: 48.3.2
```

Comment:
I am just worried about jumping 2 major versions in a dkp patch version. in the past we have created "release branches" for the helm charts in `m/charts` but I would have to remember how that process works, it was a little manual and a little risky when publishing the version

Reply:
I tried this here https://github.com/mesosphere/charts/pull/1449 backporting the fix to the KPS version we released in DKP 2.6.0 -- some more details in the PR description